src/components/FilterOptions/ClearFilters.jsx

```diff
@@ -1,21 +1,34 @@
 import { useFilter } from "contexts/";
+import CloseIcon from '@mui/icons-material/Close';
 
-const ClearFilters = () => {
-
+const ClearFilters = ({ showCloseIcon, handleChangeShowFilterDrawer }) => {
     const { filterDispatch } = useFilter();
 
     const handleClearFilters = event => {
         filterDispatch({filterType: 'CLEAR_FILTERS'});
     }
 
     return (
-        <div className="product-filters-head flex-row flex-align-end flex-justify-between mb-1-5 text-underline">
-            <h5 className="sidebar-head">Filters</h5>
-            <button className="btn btn-clear-filters btn-link" onClick={handleClearFilters}>
-                Clear Filters
-            </button>
-        </div> 
-    );
+		<div className="product-filters-head flex-row flex-align-end flex-justify-between mb-1-5 text-underline">
+			<h5 className="sidebar-head">Filters</h5>
+			<div className="flex-row flex-align-center flex-justify-center clear-filters-button-container">
+				<button
+					className="btn btn-clear-filters btn-link"
+					onClick={handleClearFilters}
+				>
+					Clear Filters
+				</button>
+				{
+                    showCloseIcon ? (
+					    <button className="btn btn-icon" onClick={() => handleChangeShowFilterDrawer(false)}>
+						    <CloseIcon fontSize="large" />
+					    </button>
+				    ) : 
```

Comment:
I didn't find anything special for correction. Clean code great naming convention. Import export is great, you are using barrel imports/exports. Great work.

Reply:
Thank you for your review!